scripts/variantstore/scripts/create_vt_bqloadjson_from_annotations.py

```diff
@@ -233,14 +248,24 @@ def make_annotated_json_row(row_position, row_ref, row_alt, variant_line, transc
         updated_dates = [] # grab the most recent
         phenotypes = [] # ordered alphabetically
         clinvar_ids = [] # For easy validation downstream
-        # Note that inside the clinvar array, are multiple objects that may or may not be the one we are looking for. We check by making sure the ref and alt are the same
+        # Note that inside the clinvar array, are multiple objects that may or may not be the one we are looking for.
+        # We check by making sure the ref and alt are the same including any reverse complements
         for clinvar_obj in clinvar_objs:
             # get only the clinvar objs with right variant and the id that starts with RCV
-            if (clinvar_obj.get("refAllele") == var_ref) and (clinvar_obj.get("altAllele") == var_alt) and (clinvar_obj.get("id")[:3] == "RCV"):
-                clinvar_ids.append(clinvar_obj.get("id"))
-                significance_values.extend([x.lower() for x in clinvar_obj.get("significance")])
-                updated_dates.append(clinvar_obj.get("lastUpdatedDate"))
-                phenotypes.extend(clinvar_obj.get("phenotypes"))
+            if (((clinvar_obj.get("refAllele") == var_ref) and (clinvar_obj.get("altAllele") == var_alt)) \
+            or ((clinvar_obj.get("refAllele") == reverse_complement(variant_line["refAllele"])) and (clinvar_obj.get("altAllele") == reverse_complement(variant_line["altAllele"])))) \
+            and (clinvar_obj.get("id")[:3] == "RCV"):
```

Comment:
might be more efficient to do the RCV check first now that the ref / alt logic has become more expensive with the reverse complement check

```suggestion
            if (clinvar_obj.get("id")[:3] == "RCV") and \
            (((clinvar_obj.get("refAllele") == var_ref) and (clinvar_obj.get("altAllele") == var_alt)) \
            or ((clinvar_obj.get("refAllele") == reverse_complement(variant_line["refAllele"])) and (clinvar_obj.get("altAllele") == reverse_complement(variant_line["altAllele"])))):
```